packages/shell/src/shell-routes.tsx

```diff
@@ -44,7 +45,27 @@ const router = createHashRouter([
       { element: <SettingsFeatureReset />, path: 'reset' },
     ],
     id: 'root',
-    loader: dbLoader(queryClient),
+    loader: async (args) => {
```

Comment:
Curious about the reason of moving this out from the loader and have it inline in the router?

Reply:
I decided to this because otherwise I needed to add the i18n code to the dbLoader function which is in the db-react package. It did not feel like the redirects or the i18n belong there and the dbLoader function should just load the db code. So making the shell handle the redirects and i18n here when the db is loaded makes more sense to me.

Reply:
Good call. Makes more sense to have it in the `shell` package.